addons/info/src/components/types/ObjectType.js

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import PrettyPropType from './PrettyPropType';
+import PropertyLabel from './PropertyLabel';
+import HighlightButton from './HighlightButton';
+
+import { TypeInfo } from './proptypes';
+
+const MARGIN_SIZE = 15;
+
+export default class ObjectType extends React.Component {
+  static propTypes = {
```

Comment:
This seems inconsistent with the rest of the codebase. I would prefer if we discuss adding proptypes like this overall before actually doing it.

If you'd like you can start a discussion about it on slack in #maintenance. For this PR, I'd refer it if we can stick to what we do everywhere else.